datafusion/sqllogictest/test_files/push_down_filter.slt

```diff
@@ -73,11 +73,29 @@ explain select uc2, column1 from  (select unnest(column2) as uc2, column1 from v
 ----
 logical_plan
 01)Projection: unnest(v.column2) AS uc2, v.column1
-02)--Filter: unnest(v.column2) > Int64(3) AND v.column1 = Int64(2)
+02)--Filter: unnest(v.column2) > Int64(3)
```

Comment:
Can you please also update the comment a few lines above to reflect the issue is fixed?

```
# Could push the filter (column1 = 2) down below unnest
# https://github.com/apache/datafusion/issues/11016
```